packages/picasso/src/SidebarItem/SidebarItemAccordion.tsx

```diff
@@ -25,16 +26,13 @@ export const SidebarItemAccordion = forwardRef<HTMLElement, Props>(
       icon,
       compact,
     } = props
-
     const classes = useStyles()
 
     const handleAccordionChange = useCallback(
       (event: ChangeEvent<{}>, isAccordionExpanded: boolean) => {
         event.stopPropagation()
 
-        if (isAccordionExpanded) {
-          expand?.(index ?? null)
-        }
+        expand?.(isAccordionExpanded ? index ?? null : null)
```

Comment:
```suggestion
        expand?.(isAccordionExpanded ? index : null)
```

I think `?? null` is redundant. `index` can be either number, undefined or null

Reply:
I see you @LashaJini and raise you with 
```suggestion
        expand?.(isAccordionExpanded && index || null)
```

because `expand()` doesn't accept `undefined` as a param type. Your suggestion will raise a TS error 🙂

Reply:
@LashaJini I had the same idea, it fails on typescript error.  It was like this before so kept it since the error disappears

Reply:
done ✅